src/test/java/org/apache/commons/text/similarity/LevenshteinDetailedDistanceTest.java

```diff
@@ -0,0 +1,354 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.commons.text.similarity;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class LevenshteinDetailedDistanceTest {
+
+    private static final LevenshteinDetailedDistance UNLIMITED_DISTANCE = new LevenshteinDetailedDistance();
+
+    @Test
+    public void testGetLevenshteinDetailedDistance_StringString() {
+    	LevenshteinResults result = UNLIMITED_DISTANCE.apply("", "");
+    	assertEquals(0, (int) result.getDistance());
+    	assertEquals(0, (int) result.getInsertCount());
+    	assertEquals(0, (int) result.getDeleteCount());
+    	assertEquals(0, (int) result.getSubstituteCount());
+ 
+    	result = UNLIMITED_DISTANCE.apply("", "a");
+       	assertEquals(1, (int) result.getDistance());
```

Comment:
@drajakumar, can you fix the indentation in the pull request? The other files also contain tabs instead of spaces. We need to sort this out before merging the code :-)